### PR TITLE
Feature/apps 3041 data set collaborations

### PR DESCRIPTION
**Pull request by solomontang**
image

### CODE REVIEW

**Review by anzioka — DISMISSED** [review]

**Review by giuliacampana — APPROVED** [review]

**Review by anzioka — APPROVED** [review]

**Review by bergieboy — APPROVED** [review]

**Review by giuliacampana — APPROVED** [review]

File: src/containers/org/OrgContainer.js [hunk 1]
```diff
@@ -78,6 +80,9 @@ const OrgContainer = ({
   const isInstalled :boolean = useSelector(selectIsAppInstalled(APPS.ACCESS_REQUESTS, organizationId));
   const collaborationsByOrganizationId = useSelector(selectCollaborationsByOrgId(organizationId));
 
+  const collaborationHref = ORG_COLLABORATIONS.replace(':organizationId', organizationId);
```
Comment (UnsungHero97) [inline src/containers/org/OrgContainer.js:83]: Please use `ORG_ID_PARAM` in the routes file.

```suggestion
  const collaborationHref = ORG_COLLABORATIONS.replace(ORG_ID_PARAM, organizationId);
```

File: src/containers/org/OrgContainer.js [hunk 2]
```diff
@@ -78,6 +80,9 @@ const OrgContainer = ({
...
+  const orgHref = ORG.replace(':organizationId', organizationId);
```
Comment (UnsungHero97) [inline src/containers/org/OrgContainer.js:84]: Please use `ORG_ID_PARAM` in the routes file.

```suggestion
  const orgHref = ORG.replace(ORG_ID_PARAM, organizationId);
```

File: src/containers/org/OrgContainer.js [hunk 3]
```diff
@@ -173,23 +174,36 @@ const OrgContainer = ({
...
-          <TabPanel show={tab === 'datasets'}>
-            <OrgDataSetsContainer organizationId={organizationId} />
-          </TabPanel>
-          <TabPanel show={tab === 'collaborations'}>
-            <CollaborationsParticipationContainer
-                collaborations={collaborationsByOrganizationId}
-                type="organization" />
-          </TabPanel>
+              value={location.pathname}>
+            <FolderTab
```
Comment (UnsungHero97) [inline src/containers/org/OrgContainer.js:182]: It's safe to delete `TabPanel` now, right?

File: src/core/router/Routes.js [hunk 4]
```diff
@@ -29,6 +29,9 @@ const ORGS :'/orgs' = '/orgs';
 // $FlowFixMe - ignoring flow because I prefer the code hints to show the value
 const ORG :'/orgs/:organizationId' = `${ORGS}/${ORG_ID_PARAM}`;
 
+// $FlowFixMe - ignoring flow because I prefer the code hints to show the value
+const ORG_COLLABORATIONS :'/orgs/:organizationId/collaborations' = `${ORGS}/${ORG_ID_PARAM}/collaborations`;
```
Comment (UnsungHero97) [inline src/core/router/Routes.js:33]: No change required, just flagging it for visibility. We're now using both `/collabs` (defined above) and `/collaborations` (defined here). I would prefer if these matched.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

**Review by anzioka — APPROVED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Pull request merged**